Replace invalid and missing URLs (4136)

stracker-phil opened the pull request:
### Description

Improve the Onboarding Wizard:

- Introduce a new `LearnMore` component
- Refactor the payment method details in `StepWelcome` and `StepPaymentMethods` into reusable micro components
- Create a new hook `usePaymentConfig` (only available for the Onboarding-screen)
- The new hook returns a config object that contains payment method details for the country and received flags (e.g. PayLater is optional)

stracker-phil committed:
✨ Prepare new LearnMore component

stracker-phil committed:
💬 Remove trailing space from URLs

stracker-phil committed:
🔀 Merge branch 'trunk'

stracker-phil committed:
🔥 Remove unused prop from BadgeBox

stracker-phil committed:
✨ Finish the LearnMore component

stracker-phil committed:
✨ Add the LearnMore link into BadgeBox

stracker-phil committed:
♻️ Improve readability of SCSS code

stracker-phil committed:
🔀 Merge branch 'trunk'

# Conflicts:
#	modules/ppcp-settings/resources/js/Components/ReusableComponents/BadgeBox.js

stracker-phil committed:
♻️ Move all learn-more links into the new prop

stracker-phil committed:
💄 Style the new LearnMore links

stracker-phil committed:
♻️ Extract repeat code into components

stracker-phil committed:
🔀 Merge branch 'trunk'

stracker-phil committed:
🎨 Add whitespace only

stracker-phil committed:
✨ New payment method components

stracker-phil committed:
♻️ Extract code into separate components

stracker-phil committed:
♻️ Extract repeat code into separate components

stracker-phil committed:
♻️ New customizable PaymentFlow component

stracker-phil committed:
✨ Add prop for each method’s learn more link

stracker-phil committed:
♻️ Shorten internal payment method name

stracker-phil committed:
✨ Add list of learn-more URLs

stracker-phil committed:
✨ Display dynamic learn-more links

stracker-phil committed:
♻️ Use new PaymentFlow in StepPaymentMethods

Narek13 commented on modules/ppcp-settings/resources/css/components/reusable-components/_badge-box.scss:
```diff
@@ -1,21 +1,14 @@
 	margin: 0 0 8px 0;
 
-	&__title {
+	.ppcp-r-badge-box__title {
```
Sorry, maybe I am missing something, but why is this changed ?

Narek13 commented on modules/ppcp-settings/resources/css/components/reusable-components/_badge-box.scss:
```diff
@@ -27,25 +20,32 @@
-			margin: 6px 0 0 0;
-			width: fit-content;
-		}
+	.ppcp-r-badge-box__title--has-image-badge .ppcp-r-title-badge--info {
```
same here

Narek13 reviewed (approved):
LGTM. Thanks

stracker-phil committed:
🔀 Merge branch 'trunk'

# Conflicts:
#	modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Components/AcdcFlow.js
#	modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Components/AcdcOptionalPaymentMethods.js
